src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -11,6 +11,7 @@ admin_orders_create:
   defaults:
     _controller: PrestaShopBundle:Admin\Sell\Order\Order:create
     _legacy_controller: AdminOrders
+    _legacy_link: AdminOrders:addorder
```

Comment:
Does it mean you enable new page already?

Reply:
It is still WIP 😅